nautobot_ssot/management/commands/elongate_interface_names.py

```diff
@@ -0,0 +1,50 @@
+"""Django Management command to update DCIM.Interface names."""
+from django.core.management.base import BaseCommand
+
+from nautobot.dcim.models import Device
+
+from netutils.interface import canonical_interface_name as netutils_elongate_interface_name
+
+
+class Command(BaseCommand):
+    """MGMT command to use netutils to update Nautobot Interface names to use long format."""
+
+    help = (
+        "Elongate DCIM.Interface names in Nautobot based off of netutils: "
+        "https://github.com/networktocode/netutils/blob/main/netutils/interface.py#L97"
+    )
+
+    def add_arguments(self, parser):  # noqa: D102
+        parser.add_argument(
+            "-d",
+            "--devices",
+            default=None,
+            help="DCIM.Device names to limit which devices have their interface names updated (comma separated).",
+        )
+
+        parser.add_argument(
+            "-l",
+            "--locations",
+            default=None,
+            help="DCIM.Location names to limit which devices have their interface names updated (comma separated).",
+        )
+
+    def handle(self, *args, **options):  # noqa: D102
+        if options.get("devices"):
```

Comment:
Thoughts on first filtering to just devices applicable for ipfabric (based on custom field but later will be solved with controller models) to prevent accidentally running the command for everything?

Reply:
Personally, I feel like Devices and Locations is a good starting point, and letting users provide feedback on what else is desired; but I can implement whatever.

Reply:
@jdrew82 what are your thoughts? I'm leaning towards constrained scoping since the management command is meant for one integration in an app not any device object in Nautobot.

Reply:
I agree on the constrained scope. It should be specific to IPFabric only.

Reply:
Why would we constrain this to a single app? The management command is the management command. Allow it to be run.

Reply:
Because the change being made is specific to one integration and we don't know how it'll impact the other ones.